src/test/verification.test.ts

```diff
@@ -2,8 +2,7 @@ import * as assert from 'assert';
 import * as vscode from 'vscode';
 
 suite('Verification', () => {
-  test.only('Program with errors has diagnostics', async () => {
-
+  test('Program with errors has diagnostics', async () => {
```

Comment:
Oof, good catch. Perhaps for later (not blocking this PR) we should add a CI check to prevent this from getting checked in.